debezium-core/src/main/java/io/debezium/metadata/ConnectorDescriptor.java

```diff
@@ -6,14 +6,31 @@
 package io.debezium.metadata;
 
 public class ConnectorDescriptor {
-    public final String id;
-    public final String name;
-    public final String version;
+    private final String id;
+    private final String name;
+    private final String className;
+    private final String version;
 
-    public ConnectorDescriptor(String id, String name, String version) {
+    public ConnectorDescriptor(String id, String name, String className, String version) {
```

Comment:
@rk3rn3r, do we actually need the separation of `AbstractConnectorMetadata` and `ConnectorDescriptor`? I feel it could all be just on one "flat" class.

Reply:
Yes, I think there was a time where `ConnectorDescriptor` was used as a JSON model entity to return it as part of the schema. But this really seems obsolete now. :+1:

Reply:
Ok, so we agreed: we can get rid of this `ConnectorDescriptor` class and pull up everything into `ConnectorMetadata`.